3rdparty/lua-5.1.3/src/loslib.c

```diff
@@ -19,6 +23,7 @@
 #include "lauxlib.h"
 #include "lualib.h"
 
+extern char **environ;
```

Comment:
does `environ` need to be extern? since it seems to be used only in `os_execute` perhaps it can be declared as `static char **environ` in that function?